### DISCUSSION TITLE
what's a ip address and where can i find it?

### DISCUSSION THREAD

**Discussion by kokodemon** [MinicraftPlus/minicraft-plus-revived#649] [Q&A]

**Comment by kokodemon** [comment]
i logged in but it asked for somekinda ip address and localhost. Could somebody explain me?

**Comment by chrisj42** [comment]
an "ip address" is how you identify different computers across the internet. "localhost" is a special ip address that means "the current computer". So, if you are running a server and want to connect to it from the very same computer, you could connect to "localhost".

Does that help?

**Comment by kokodemon** [comment]
> an "ip address" is how you identify different computers across the internet. "localhost" is a special ip address that means "the current computer". So, if you are running a server and want to connect to it from the very same computer, you could connect to "localhost".
> 
> Does that help?

Thank u very muc!!!! It helped a lot! =3

**Comment by chrisj42** [comment]
awesome, glad I could help. :)
